src/components/Board/Board.tsx

```diff
@@ -147,7 +148,7 @@ export const BoardComponent = ({children, currentUserIsModerator, moderating}: B
   return (
     <>
       <style>{`.board { --board__columns: ${columnsCount} }`}</style>
-
+      <ReactTooltip type={localStorage.getItem("theme") === "dark" ? "dark" : "light"} place="bottom" delayShow={500} multiline />
```

Comment:
localStorage doesn't properly reflect the current theme if it is set to "auto", which leads to the tooltips being white when theme is set to auto and the OS setting is set to dark. Maybe you can access the "theme" property on the <html> element?